client/my-sites/plugins/plugin-details.jsx

```diff
@@ -226,6 +227,7 @@ function PluginDetails( props ) {
 			<SidebarNavigation />
 			<QueryEligibility siteId={ selectedSite?.ID } />
 			<QueryProductsList persist />
+			<QueryJetpackPlugins siteIds={ siteIds } />
```

Comment:
Let's conditionally query the plugin only if  `isJetpack === true` (atomic or self hosted).

Reply:
I don't think it is needed as the `siteIds` values come from the `getSelectedOrAllSitesWithPlugins` which already validates if it is jetpack or not. 

https://github.com/Automattic/wp-calypso/blob/d071c4e7d7c0f2d32981409670e36c75c74f784f/client/state/selectors/get-selected-or-all-sites-with-plugins.js#L16-L23